src/avram/base_query_template.cr

```diff
@@ -39,6 +39,10 @@ class Avram::BaseQueryTemplate
           end
         {% end %}
 
+        {% if columns.map(&.[:name].stringify).includes?("updated_at") %}
+          _changes[:updated_at] = Time.adapter.to_db(Time.utc) if updated_at == IGNORE
```

Comment:
This reads a little confusing... It looks like it's saying "Only change updated_at if I have updated_at set to ignore". Is it just me, or should this not work? :thinking:  I see specs passed so it must be right.

Reply:
Yes. The idea is that if `update_at` is passed in explicitly, then use that. So only set `updated_at` if it was not passed in, in which case its value would be `IGNORE`.

This means `PostQuery.new.update(title: "New title")` would set `updated_at` to current time, but `PostQuery.new.update(title: "New title", updated_at: 2.days.ago)` would set the `updated_at` to 2 days ago. (I should probably add a spec to cover this case)

Don't merge yet, though. I think the condition should be `if updated_at.is_a?(Avram::Ignore)`. Classes compare by identity, so one `Avram::Nothing.new` would not necessarily equal another `Avram::Nothing.new`. I'll add one more commit to address this.